### PR TITLE
feat(FEC-8872): add provider check for entry status import and converting

### PR DESCRIPTION
**Pull request by RoyBregman**
### Description of the Changes

Added a check in OVP provider to check status 0 or 1 from the multiRequest response and throw a specific new sub error 2002 of error 7002

solves FEC-8872

### CheckLists

- [X] changes have been done against master branch, and PR does not conflict
- [X] new unit / functional tests have been added (whenever applicable)
- [X] test are passing in local environment
- [X] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/k-provider/ovp/provider.js [hunk 1]
```diff
@@ -128,6 +130,18 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
...
   }
 
+  /**
+   * Checks media is ready for playback (not being imported or converted)
+   * @param {MediaEntry} mediaEntry - the media entry info
+   * @returns {void}
+   */
+  _verifyMediaStatus(mediaEntry: MediaEntry) {
+    if ([KalturaMediaEntry.EntryStatus.IMPORT, KalturaMediaEntry.EntryStatus.PRECONVERT].includes(mediaEntry.status)) {
+      throw new Error(Error.Severity.CRITICAL, Error.Category.SERVICE, Error.Code.MEDIA_STATUS_IN_PREPARATION, {
```
Comment (OrenMe) [inline src/k-provider/ovp/provider.js:140]:
```suggestion
      throw new Error(Error.Severity.CRITICAL, Error.Category.SERVICE, Error.Code.MEDIA_NOT_READY, {
```

File: src/k-provider/ovp/provider.js [hunk 2]
```diff
@@ -128,6 +130,18 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
...
+        messages: `Status of entry id ${mediaEntry.id} is ${mediaEntry.status} and is still being imported or converted`
```
Comment (OrenMe) [inline src/k-provider/ovp/provider.js:141]: include the status as data `{status}`

File: src/k-provider/ovp/provider.js (hunk 1 above)
Comment (RoyBregman) [reply src/k-provider/ovp/provider.js]: done

File: src/k-provider/ovp/provider.js (hunk 2 above)
Comment (RoyBregman) [reply src/k-provider/ovp/provider.js]: done

File: src/util/error/code.js [hunk 3]
```diff
@@ -66,7 +66,12 @@ const Code: CodeType = {
   /**
    * The provider doesn't implement the called api
    */
-  METHOD_NOT_IMPLEMENTED: 3002
+  METHOD_NOT_IMPLEMENTED: 3002,
+
+  /**
+   * The server responded with status import or pre convert
+   */
+  MEDIA_STATUS_NOT_READY: 3003
```
Comment (OrenMe) [inline src/util/error/code.js:74]: should be 2xxx error, move to SERVICE address range

Comment (OrenMe) [reply src/util/error/code.js:74]: @RoyBregman ?

Comment (RoyBregman) [reply src/util/error/code.js]: fixed and pushed

**Pull request merged**